### ISSUE TITLE
auro-comments

### ISSUE THREAD

**Issue by braven112** [AlaskaAirlines/auro#253]
What is the best way to handle comments when used with form fields?  Do we need a new component or a variation of an existing one?

James Tsai is working on updating some of the forms for promo pages that content and loyalty teams supports.  They are trying to add auro input to these pages in phase 1.  In phase 2 they need a way to allow guests to add comments to their form submission. 

We need a blueprint for this body of work.  To start, the blueprints only needs to accomplish the goals around this initiative but of course consider other common use cases.

Examples:
https://www.delldesignsystem.com/components/text-area/
https://atlassian.design/components/textarea/examples
https://shoelace.style/components/textarea

**Comment by blackfalcon** [comment]
This element raises a few interesting questions.

1. the label for this element is going to be difficult to make fit on our currently animated interaction. [Material UI](https://material.io/components/text-fields#outlined-text-field) has 'interesting' ways of addressing this. 
2. Should `auro-textarea` be a type in `auro-input`? 
3. If `auro-textarea` is to be an individual element, do we make the label and help text individual elements?

**Comment by leeejune** [comment]
@blackfalcon I was wondering if the "grip" and "scroll bar" on text areas are customizable? (the comment text box in Github is an example). It seems like there are no customizations in other text areas online.

**Comment by blackfalcon** [comment]
@leeejune I don't believe that those are customizable. Besides we pretty much take a position of not customizing core UIs that browsers already provide as that will create confusion with users.

**Comment by leeejune** [comment]
https://www.figma.com/file/Ruztq0h7kDYrpOWSfwTWyh/Text-Area?node-id=322%3A298

Completed blueprint.

**Comment by blackfalcon** [comment]
Looking at the examples provided, I have to admit I am not a fan of things. Playing around with Spectrum the addition of additional padding to the right when an icon is dropped in is not optimal. 

What about something where there is a little more padding at the top and the bottom and then we can reserve space at the bottom for the placement of an icon?

**Comment by leeejune** [comment]
Here are explorations I did. I will also ask the UI team for their perspective on this!

The top part lays out all the options that was discussed during our standup. The bottom shows examples of what it would look like with our Input and Dropdown component.
